client/templates/includes/scripts.njk

```diff
@@ -4,6 +4,7 @@
 <script src="{{ assetPath }}js/bundle.js"></script>
 
 <!--  Init GDS and MOJ toolkits -->
+<script src="{{ assetPath }}js/govuk/all.js"></script>
```

Comment:
Is this an unrelated change? Probably not actually a concern, but theoretically should be separate?

Reply:
I missed this when I updated the gds toolkit yesterday :D removed by mistake but added as part of this fix